app/controllers/games_controller.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+# Comment necessary for Rubocop
+class GamesController < ApplicationController
+  before_action :set_game, only: %i[show update destroy]
+
+  # GET /games
+  def index
+    @games = Game.all
+
+    render json: @games
+  end
+
+  # GET /games/1
+  def show
+    render json: render_game_with_state_and_player(@game)
+  end
+
+  # POST /games
+  def create
+    Game.delete_all
+    new_game = TicTacToeLogic::GameBoardData.new
+    @game = Game.new({ "board": new_game })
+
+    if @game.save
+      render json: render_game_with_state_and_player(@game)
+    else
+      render json: @game.errors, status: :unprocessable_entity
+    end
+  end
+
+  # POST '/submit_move/:move'
+  def submit_move
+    @game = Game.find(1)
+    @game.board.play_move(params[:move].to_i)
+
+    if @game.save
+      render json: render_game_with_state_and_player(@game)
+    else
+      render json: @game.errors, status: :unprocessable_entity
+    end
+  end
+
+  # DELETE /games/1
+  def destroy
+    @game.destroy
```

Comment:
Having `@game` be an instance variable seems dangerous. Here you have not set @game to anything before calling `.destroy`  So what ever `@game` was set to on the last call will get deleted here.

Reply:
This is using that set_game method. So it will set @game to the id passed in. 
this line at the top is being used to call that set_game method: 
`before_action :set_game, only: %i[show update destroy]`

Do I still need to change something?

Reply:
nevermind. I didnt' see the `before_action`

Reply:
👍🏽